examples/service_envoy/listeners.pb

```diff
@@ -0,0 +1,16 @@
+listeners {
```

Comment:
It's a ListenerDiscoverResponse, see https://github.com/lyft/envoy-api/blob/master/api/lds.proto#L39.